apps/cowswap-frontend/src/common/pure/ButtonSecondary/index.tsx

```diff
@@ -11,7 +11,7 @@ export const ButtonSecondary = styled.button<{ padding?: string; minHeight?: str
   box-shadow: none;
   border-radius: 12px;
   position: relative;
-  transition: background 0.2s ease-in-out;
+  transition: background 0.1s ease-in-out;
```

Comment:
I think we should move the time into a const, like a `UI.TRANSITION_TIME`

Reply:
Yes agreed

Reply:
Fixed